WooCommerce/Classes/ViewRelated/Dashboard/Settings/About/WooAboutScreenConfiguration.swift

```diff
@@ -0,0 +1,210 @@
+import UIKit
+import AutomatticAbout
+import SafariServices
+
+class WooAboutScreenConfiguration: AboutScreenConfiguration {
```

Comment:
nit: can this class be final?
```suggestion
final class WooAboutScreenConfiguration: AboutScreenConfiguration {
```